main.cpp

```diff
@@ -331,9 +349,21 @@ void GuitarRoom::turnRatioKnob(int newRatio)
     std::cout << "New compression ratio: " << guitarPedal.initRatio << std::endl;
 }
 
+void GuitarRoom::modifyPickupsThis(int numPickUps)
+{
+    this -> eGuitar.numPickUps = numPickUps;
+    std::cout << "New number of guitar Pickups: " << eGuitar.numPickUps << std::endl;
+}
+
+void GuitarRoom::turnRatioKnobThis(int ratio)
+{
+    this->guitarPedal.initRatio = ratio;
+    std::cout << "New compression ratio: " << guitarPedal.initRatio << std::endl;
+}
```

Comment:
don't put spaces around the `->` operator.
your std::cout statements need to show usage of `this->`.
- [ ] revise